### DISCUSSION TITLE
打开过滤大陆域名/IP开关后无法连接

### DISCUSSION THREAD

**Discussion by Dameko27r** [trojan-gfw/igniter#377] [Suspicious Problems]
安卓9.0 关闭该选项的情况下可以连接。上一版本可正常使用。

**Comment by Dameko27r** [comment]
I mean alpha14 is wroking

**Comment by wongsyrone** [comment]
We only support the latest version, it's igniter 0.10.2-beta at the moment.

**Comment by jy1989** [comment]
0.10.2-beta 最新版也有一样的问题，打开开关之后，无法连接

**Comment by TchaikovDriver** [comment]
> 0.10.2-beta 最新版也有一样的问题，打开开关之后，无法连接

麻烦提供一下相关的log

**Comment by pullopen** [comment]
> 
> 
> 0.10.2-beta 最新版也有一样的问题，打开开关之后，无法连接

同样的问题，0.10.2-beta版。
